src/index.css

```diff
@@ -201,7 +201,7 @@ th {
 .member {
   height: 140px;
   width: 140px;
-  padding: .75em;
+  /* padding: .75em; */
```

Comment:
I'm assuming this is still WIP but I might refactor this and the related avatar styles to more clearly separate the styles per class. I also like putting the word "avatar" on the class name to more immediately make clear what precisely the style is referring to:

```
.avatar {
  border: 1px solid #707070;
  background-color: #f5b742;
}

.admin-avatar {
  height: 140px;
  width: 140px;
}

.member-avatar {
  height: 100px;
  width: 100px;
  border-radius: 50%;
}
```

You'll of course want to apply both of these styles to the relevant avatar rows, .e.g:

`<img className='avatar admin-avatar' src={member.imgUrl} alt={`beautiful headshot of ${member.name}.`} />`

Reply:
At time of this comment it looks like this:

```
.admin,
.member {
  height: 140px;
  width: 140px;
  /* padding: .75em; */
  border: 1px solid #707070;
  background-color: #f5b742;
}

.member {
  height: 100px;
  width: 100px;
  border-radius: 50%;
}
```

The issue here is that the initial assignment of the height/width to `.member` is overwritten in the next style block. In my previous comment I've refactored these two blocks into three styles: one for the styles that are common among both admins and members, and then one each for the styles that are unique to each.